Text/PresidioPIIAnonymization/powerskill/requirements.txt

```diff
@@ -0,0 +1,14 @@
+fastapi~=0.63.0
+python-dotenv~=0.17.0
+uvicorn~=0.13.4
+ObjDict==0.4.4
+azure-core==1.12.0
+azure-search==1.0.0b2
+azure-search-documents==11.1.0
+presidio-analyzer==2.2.0
+presidio-anonymizer==2.2.0
+pydantic~=1.7.3
+starlette~=0.13.6
+requests~=2.25.1
+numpy~=1.20.2
+scipy~=1.6.2
```

Comment:
I don't see any usage in some of the stuff, like numpy, scipy. Why are those needed?

Reply:
it was automatically added... removed and tested. Also fixed the versions.
Good comments. thank you.